pkg/cloud/services/route53/route53.go

```diff
@@ -103,11 +112,13 @@ func (s *Service) ReconcileRoute53() error {
 				}
 				_, err = s.Route53ResolverClient.AssociateResolverRule(i)
 				if err != nil {
-					return errors.Wrapf(err, "failed to assign resolver rule %s to VPC %s", *rule.Name, s.scope.VPC())
+					s.scope.Error(err, "failed to assign resolver rule to VPC", "ruleName", *rule.Name, "vpc", s.scope.VPC())
```

Comment:
We want to keep associating rules even when associating one of them fails for any reason.